modules/http/src/test/scala/higherkindness/mu/rpc/http/GreeterRestTests.scala

```diff
@@ -48,25 +44,21 @@ class GreeterRestTests extends RpcBaseTestSuite with BeforeAndAfter {
   val Fs2ServicePrefix   = "Fs2Greeter"
   val MonixServicePrefix = "MonixGreeter"
 
-  implicit val ec                   = monix.execution.Scheduler.Implicits.global
+  implicit val ec                   = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Do you know who requires an implicit `ExecutionContext`? With the `ContextShift` it shouldn't be necessary to be implicit.

Reply:
Merged. I’m clarifying this in my current branch. Thanks.